tools/buildhelper/buildhelper.go

```diff
@@ -32,13 +35,56 @@ import (
 const (
 	develBranchName     = "devel"
 	releaseBranchPrefix = "release-"
+
+	versionFileName  = "Makefile"
+	versionTagPrefix = `VERSION ?= `
```

Comment:
A prefix with a space ` ` in the name?

Reply:
this is admittedly fragile, but I'd like to avoid going down the full makefile parsing route or regexp parsing route just yet